include/deal.II/base/mpi.h

```diff
@@ -267,19 +267,6 @@ namespace Utilities
     class MPI_InitFinalize
     {
     public:
-
-      /**
-       * Constructor. Takes the arguments from the command line (in case of
-       * MPI, the number of processes is specified there), and sets up a
-       * respective communicator by calling <tt>MPI_Init()</tt>. This
-       * constructor can only be called once in a program, since MPI cannot be
-       * initialized twice.
-       *
-       * This constructor sets max_num_threads to 1 (see other constructor).
-       */
-      MPI_InitFinalize (int    &argc,
-                        char ** &argv) /*DEAL_II_DEPRECATED*/;
-
```

Comment:
I didn't see this before but this function was never really deprecated :-/ This is just a comment, it never showed up in the documentation (see http://dealii.org/8.2.0/doxygen/deal.II/classUtilities_1_1MPI_1_1MPI__InitFinalize.html). It should have been tagged as @deprecated to appear in the documentation. Should we put it back or just put that as an incompatible change? This change is going to break a lot of people's code since it's the constructor that we used the most often in the tutorials.